src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="post"
+      class="form form--large"
+      data-qa="big"
+    >
+      <input
+        class="search__bar search__bar--large"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
+    </form>
+    <form
```

Comment:
It is a good idea to add blank line between multiline siblings
```suggestion
    </form>
    
    <form
```